qiita_pet/handlers/software.py

```diff
@@ -72,13 +72,23 @@ def _default_parameters_parsing(node):
         #                   output_type: output_node_name}, ...}
         # for easy look up and merge of output_names
         main_nodes = dict()
-        for x, y in graph.edges:
+        for i, (x, y) in enumerate(graph.edges):
```

Comment:
Just to confirm, the 'i' coming from enumerate is meant to identify the very first loop iteration only? I guess if it depends on having x and y defined then it's overall the cleanest representation.

Reply:
This is correct: i is enumerate and is just to figure out if the first loop.